modules/ROOT/pages/sync/adding-sync-to-your-app.adoc

```diff
@@ -65,7 +79,52 @@ Apollo will automatically create a file `MyNotesQuery`, `CreateNoteMutation` and
 [role="secondary"]
 .iOS
 ****
-Coming soon
+1. Create a file `schema.json` with your GraphQL schema.
+2. Create a file with a name that corresponds to your business objects and a `.graphql` extension.
+3. Add all your queries/mutations/subscriptions to the `.graphql` file.
+4. Add a new run script build phase to your target (should be above Compile Sources phase).
+5. Copy the script below to the phase:
+
+```
+APOLLO_FRAMEWORK_PATH="$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1)"
```

Comment:
Apollo changes this stuff every release. I think we should point people to their documentation rather than copy this.

Reply:
Thanks for review. I will replace the script with a link to the official docs.

For the steps above I would leave them as they are as the example is mentioned in later parts of this doc (queries/mutations/...), is that ok? Or should we add more links to official docs [1]?

[1] https://www.apollographql.com/docs/ios/installation.html

Reply:
I think that all queries are fine etc. Just initial setup script is better to be linked upstream.